Test/doofiles/Test4.dfy

```diff
@@ -0,0 +1,8 @@
+// RUN: %baredafny build -t:lib --use-basename-for-filename "%s" --output "%S/test2" > "%t"
+// RUN: %baredafny resolve --use-basename-for-filename "%s"  "%S/test2.doo" >> "%t"
+// RUN: %diff "%s.expect" "%t"
+// XFAIL: *
```

Comment:
Again I won't block on it, but couldn't we use `!` or `%exits-with` to be more precise about what fails? I'd generally prefer to avoid `XFAIL`.

Reply:
This is not really an expected failure -- it is a bug waiting to be fixed, as described in the relevant issue. I agree that a real XFAIL is better written with %exists-with (which was the purpose of developing %exits-with in the first place).